tests/php/test_class.json-api-jetpack-endpoints.php

```diff
@@ -159,6 +159,72 @@ public function test_Jetpack_API_Plugins_Install_Endpoint() {
 		$this->rmdir( $the_real_folder );
 	}
 
+	/**
+	 * @author wpbenyamin
+	 * @covers Jetpack_JSON_API_Export_Endpoint
+	 * @requires PHP 5.3.2
+	 */
+	public function test_Jetpack_JSON_API_Export_Endpoint() {
+		$endpoint = new Jetpack_JSON_API_Plugins_Install_Endpoint( array(
+			'method' => 'POST',
```

Comment:
I'm not familiar with testing endpoints in Jetpack - is it expected we need to duplicate all of the fields from the endpoint definition?